ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/ImagesAtMostRiskTable.tsx

```diff
@@ -0,0 +1,111 @@
+    riskPriority: 'Risk priority',
+    criticalCves: 'Critical CVEs',
+    importantCves: 'Important CVEs',
+};
+
+function linkToImage(id: string) {
+    return `${vulnManagementPath}/image/${id}#image-findings`;
+}
+
+function ImagesAtMostRiskTable({ imageData: { images }, cveStatusOption }: ImagesAtMostRiskProps) {
+    return (
+        <TableComposable aria-label="Images at most risk" variant="compact" borders={false}>
+            <Thead>
+                <Tr>
+                    <Th width={35} className="pf-u-pl-0">
+                        {columnNames.imageName}
+                    </Th>
+                    <Th className="pf-u-text-align-center-on-md">{columnNames.riskPriority}</Th>
+                    <Th>{columnNames.criticalCves}</Th>
+                    <Th className="pf-u-pr-0">{columnNames.importantCves}</Th>
+                </Tr>
+            </Thead>
+            <Tbody>
+                {images.map(({ id, name, priority, vulnCounter }) => (
+                    <Tr key={id}>
+                        <Td className="pf-u-pl-0" dataLabel={columnNames.imageName}>
+                            <Link
+                                to={linkToImage(id)}
+                                scroll={(el: HTMLElement) =>
+                                    // TODO This is a heavy handed way to scroll to the CVE section which is loaded on
+                                    // the target image page asynchronously. Without a delay, following data loads
+                                    // scroll the target element back off the screen.
+                                    setTimeout(() => el.scrollIntoView({ behavior: 'smooth' }), 500)
+                                }
+                            >
+                                <Tooltip content={<div>{name.fullName}</div>}>
+                                    <span title={name.fullName}>{middleTruncate(name.remote)}</span>
+                                </Tooltip>
+                            </Link>
+                        </Td>
+                        <Td
+                            className="pf-u-text-align-center-on-md"
+                            dataLabel={columnNames.riskPriority}
+                        >
+                            {priority}
+                        </Td>
+                        <Td dataLabel={columnNames.criticalCves}>
+                            <SecurityIcon
+                                className="pf-u-display-inline pf-u-mr-xs"
+                                color={severityColors.critical}
+                            />
+                            <span>
+                                {cveStatusOption === 'Fixable'
+                                    ? `${vulnCounter.critical.fixable} fixable`
+                                    : `${vulnCounter.critical.total} CVEs`}
+                            </span>
+                        </Td>
+                        <Td className="pf-u-pr-0" dataLabel={columnNames.importantCves}>
```

Comment:
Too bad, so sad, that security icon at the left (which is same for every cell in column) requires left alignment instead of right alignment convention for numeric data:

| left | right |
| :--- | ---: |
| 123 fixable | 123 fixable |
| 12 fixable | 12 fixable |
| 1 fixable | 1 fixable |

At least it does not have center alignment

Reply:
I can bring the alignment concerns up during a sync and potentially have this as a follow up in the 72 release. There are plans for many more new widgets in the future so the sooner we get ahead of this the better.